### PR TITLE
feat(otel): log notice to STDOUT when using OTel

### PR DESCRIPTION
**Pull request by 10xLaCroixDrinker**
## Description

Rather than simply leaving an empty console on startup when using OTel, send a single message informing the user that startup was successful and that logs are going to OTel.

## Motivation and Context

DX improvement

## How Has This Been Tested?

Unit tests and running locally validating that log is present

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)
- [ ] Dependency update
- [ ] Security update

## Checklist:

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [x] These changes should be applied to a maintenance branch.
- [ ] This change requires cross browser checks.
- [ ] Performance tests should be ran against the server prior to merging.
- [ ] This change impacts caching for client browsers.
- [ ] This change impacts HTTP headers.
- [ ] This change adds additional environment variable requirements for One App users.
- [ ] I have added the Apache 2.0 license header to any new files created.

## What is the Impact to Developers Using One App?

More clear what's happening with the logs from the application console when using OTel for logging

### CODE REVIEW

**Review by JAdshead — DISMISSED** [review]

**Review by smackfu — DISMISSED** [review]

**Comment by 10xLaCroixDrinker** [comment]
Need to add another log for error scenario, failure to start

**Review by giulianok — DISMISSED** [review]
lgtm

**Comment by 10xLaCroixDrinker** [comment]
> Need to add another log for error scenario, failure to start

done in [eb166b0](https://github.com/americanexpress/one-app/pull/1215/commits/eb166b004c3b66afd6cbce0abfa1a3eb81716c4a)

**Review by Matthew-Mallimo — APPROVED** [review]

File: src/server/utils/logging/config/otel.js [hunk 1]
```diff
@@ -44,6 +45,13 @@ const { buildVersion: version } = readJsonFile('../../../.build-meta.json');
 //   });
 // }
 
+// setGlobalErrorHandler((err) => {
+//   process.stderr.write('OTel Error one-app: ');
+//   process.stderr.write(err);
+//   loggingErrorHandler(err);
+//   process.exit(1);
+// });
```
Comment (Matthew-Mallimo) [inline src/server/utils/logging/config/otel.js:53]: Can we remove this instead of leaving it commented

Comment (10xLaCroixDrinker) [reply src/server/utils/logging/config/otel.js:53]: Dang, committed this on accident 🤦🏻

**Review by Matthew-Mallimo — APPROVED** [review]

**Review by giulianok — APPROVED** [review]

**Pull request merged**